### PR TITLE
#70 User page now shows every registered for the company user except for admins

### PR DESCRIPTION
**Pull request by Werfit**

### CODE REVIEW

File: frontend/src/components/app/app.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import UserRouting from './user-routing';
 import './app.scss';
 
 const App: React.FC = () => {
-  const isAdmin = false;
+  const isAdmin = true;
```
Comment (nikita-remeslov) [inline frontend/src/components/app/app.tsx]: switch to false

**Review by nikita-remeslov — APPROVED** [review]
Fix comment

File: backend/src/api/routes/user-route.ts [hunk 2]
```diff
@@ -19,12 +23,17 @@ router.put(
...
       req.companyId,
     ),
   ),
 );
 
+router.get(
+  '/',
+  run((req: Request) => getCommonUserList(req.companyId)),
+);
+
```
Comment (aleksandr-kuharenko) [inline backend/src/api/routes/user-route.ts]: move to /api/company/users

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]

File: backend/src/api/routes/user-route.ts [hunk 3]
```diff
@@ -2,7 +2,11 @@ import { Request, Router } from 'express';
 import multer from 'multer';
 
 import { run } from '../../common/helpers/route.helper';
```
Comment (Pavel1066) [inline backend/src/api/routes/user-route.ts]: change to '~/common/helpers/route.helper';

File: backend/src/api/routes/authentication-route.ts [hunk 4]
```diff
@@ -3,10 +3,9 @@ import { Request, Router } from 'express';
 import { run } from '../../common/helpers/route.helper';
```
Comment (Pavel1066) [inline backend/src/api/routes/authentication-route.ts]: change to '~/common/helpers/route.helper';

**Review by Pavel1066 — CHANGES_REQUESTED** [review]

**Review by aleksandr-kuharenko — APPROVED** [review]

**Review by Pavel1066 — APPROVED** [review]

**Pull request merged**